libs/server/Servers/GarnetServerOptions.cs

```diff
@@ -369,7 +369,7 @@ public GarnetServerOptions(ILogger logger = null) : base(logger)
         /// <summary>
         /// Get KVSettings for the main store log
         /// </summary>
-        public KVSettings<SpanByte, SpanByte> GetSettings(ILogger logger, out INamedDeviceFactory logFactory)
+        public KVSettings<SpanByte, SpanByte> GetSettings(ILoggerFactory loggerFactory, ILogger logger, out INamedDeviceFactory logFactory)
```

Comment:
Probably cleaner to instantiate the logger within this method as you're already supplying the logger factory. Also probably a good idea to either rename the out parameter or add an XML comment, since it might be confusing (loggerFactory vs. logFactory).

Reply:
Added an XML comment